From 3988ba1797ffabcb4a14010c876b1343389cc1ab Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Tue, 26 Feb 2019 09:42:04 -0800
Subject: [PATCH] Proper path for rrdcached service files.

---
 etc/Makefile.am | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/etc/Makefile.am b/etc/Makefile.am
index ab763dc..e2b70b4 100644
--- a/etc/Makefile.am
+++ b/etc/Makefile.am
@@ -3,6 +3,6 @@ EXTRA_DIST = rrdcached-default-redhat rrdcached-init-redhat rrdcached-default-ls
 
 if HAVE_SYSTEMD
 systemdsystemunit_DATA = \
-	etc/rrdcached.socket \
-	etc/rrdcached.service
-endif
\ No newline at end of file
+	rrdcached.socket \
+	rrdcached.service
+endif
-- 
2.20.1

